stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -740,13 +741,13 @@ internal class ChatDomainImpl internal constructor(
 
     suspend fun storeStateForChannels(channelsResponse: List<Channel>) {
         val users = mutableMapOf<String, User>()
-        val configs: MutableMap<String, Config> = mutableMapOf()
```

Comment:
Similar question here, was this a `Map` for no reason, do we never see duplicate channel types here that would override each other in the loop below?

Reply:
For me, it is strange to have doubling here. Anyway if we had then it would get overridden in the Database. So I just changed the collection type to set.